### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by orinevares**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/copyright-edit-request.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+name: Copyright Edit Request
```
Comment (dlevineBC) [inline .github/ISSUE_TEMPLATE/copyright-edit-request.md]: This wouldn't be considered copyright edit, that's a legal thing. Should be called "Content Edit Request"

File: .github/ISSUE_TEMPLATE/component-bug-report.md [hunk 2]
```diff
@@ -1,6 +1,9 @@
 ---
 name: Component Bug report
 about: Create a report to help us improve the BC GOV Design System Components
+title: ''
```
Comment (dlevineBC) [inline .github/ISSUE_TEMPLATE/component-bug-report.md]: Why were these 3 new attributes added?

File: .github/ISSUE_TEMPLATE/component-bug-report.md [hunk 3]
```diff
@@ -1,6 +1,9 @@
 ---
 name: Component Bug report
 about: Create a report to help us improve the BC GOV Design System Components
```
Comment (dlevineBC) [inline .github/ISSUE_TEMPLATE/component-bug-report.md]: "Create a bug report..."

File: .github/ISSUE_TEMPLATE/copyright-edit.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+---
+name: Copyright Edit
```
Comment (dlevineBC) [inline .github/ISSUE_TEMPLATE/copyright-edit.md]: This also shouldn't say "copyright". And what is the difference between this template and the "Content Edit Request"? These seem like duplicates

**Comment by orinevares** [comment]
@dlevineBC not sure why things are getting duplicated. I'm going to close this PR and try again to reset it.

**Pull request closed without merging**